### PR TITLE
b/wpc-423-ui-fixes

### PR DESCRIPTION
**Pull request by HanaAyman**
This pull request is to fix some UI bugs. 

1. Language Icon on the bottom left of the editor should only be blue when hovering.
2.  Plus Icon should be behind the actions buttons in the articles and pages.
3. There should be some margin between the blocks and the buttons in the articles and pages.

### CODE REVIEW

File: packages/editor/src/client/atoms/editorTemplate.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function EditorTemplate({children, navigationChildren}: EditorTemplatePro
           display: 'flex',
           position: 'sticky',
           top: 0,
-          zIndex: 3,
+          zIndex: 10,
```
Comment (bishoymelek) [inline packages/editor/src/client/atoms/editorTemplate.tsx:25]: Why have you changed this one? :eyes:

File: packages/editor/src/client/base.tsx [hunk 2]
```diff
@@ -23,6 +23,8 @@ import {
 
 import {useTranslation} from 'react-i18next'
 
+import './global.less'
```
Comment (bishoymelek) [inline packages/editor/src/client/base.tsx:26]: It's already imported in `app.tsx` which uses base.tsx .. I guess you don't need to import it again in base component

**Review by bishoymelek — CHANGES_REQUESTED** [review]
Helpful points to align styling! :100:  just two points to ask about.

File: packages/editor/src/client/atoms/editorTemplate.tsx (hunk 1 above)
Comment (HanaAyman) [reply packages/editor/src/client/atoms/editorTemplate.tsx:25]: This one is to make the plus buttons appear behind the metadata and the save buttons when scrolling.

Comment (bishoymelek) [reply packages/editor/src/client/atoms/editorTemplate.tsx:25]: You mean disappear?

Comment (HanaAyman) [reply packages/editor/src/client/atoms/editorTemplate.tsx:25]: yes as it currently passes on top of the buttons.
image

**Review by roosnic1 — APPROVED** [review]
👍🏾

File: packages/editor/src/client/routes/articleEditor.tsx [hunk 3]
```diff
@@ -408,7 +408,7 @@ export function ArticleEditor({id}: ArticleEditorProps) {
             <NavigationBar
               leftChildren={
                 <IconButtonLink
-                  style={{marginTop: '4px'}}
+                  style={{marginTop: '4px', marginBottom: '20px'}}
```
Comment (bishoymelek) [inline packages/editor/src/client/routes/articleEditor.tsx:411]: Do we need this margin? won't it be affected by the one on line 422?

**Review by bishoymelek — COMMENTED** [review]
Just very small point

File: packages/editor/src/client/base.tsx [hunk 4]
```diff
@@ -214,6 +214,7 @@ export function Base({children}: BaseProps) {
                   trigger="click"
                   renderTitle={() => (
                     <Icon
+                      className="language-selector"
```
Comment (penk85) [inline packages/editor/src/client/base.tsx:217]: What's the advantage of having the icon blue only on hover? For consistency, can we have the same styling on the settings icon?

File: packages/editor/src/client/atoms/editorTemplate.tsx (hunk 1 above)
Comment (penk85) [inline packages/editor/src/client/atoms/editorTemplate.tsx:25]: What browser is it you use where it overlaps? It doesn't overlap on my screen.

Comment (HanaAyman) [reply packages/editor/src/client/atoms/editorTemplate.tsx:25]: I use both google chrome and Microsoft edge and for me it overlapped on both.

Comment (penk85) [reply packages/editor/src/client/atoms/editorTemplate.tsx:25]: Oh I see it now. Good catch :+1:

**Review by penk85 — APPROVED** [review]
:+1:

**Review by bishoymelek — APPROVED** [review]

**Review by roosnic1 — APPROVED** [review]
👍🏾

**Pull request merged**